morepath/path.py

```diff
@@ -142,9 +141,9 @@ def register_path_variables(self, model, func):
         :param func: function that gets a model instance argument and
           returns a variables dict.
         """
-        self.reg_registry.register_function(generic.path_variables,
-                                            func,
-                                            obj=model)
+        self.reg_registry._get_path_variables.register(
```

Comment:
Again on my branch thanks to ``app_class_arg`` and the dispatch objects being directly on app class, the whole scope registry notion doesn't exist. I think it makes for easier reading, as there aren't two places where something like ``_get_path_variables`` can exist - no decision needs to be made about which one you can use depending on whether you register or call it.